app/[...slug]/page.tsx

```diff
@@ -0,0 +1,55 @@
+import { getStoryblokApi, ISbStoriesParams, StoryblokClient } from '@storyblok/react/rsc';
+import StoryblokStory from '@storyblok/react/story';
+import { resolveRelations } from '@/utilities/resolveRelations';
+
+const activeEnv = process.env.NODE_ENV || 'development';
+
+export default async function Page({ params }) {
+  const { data } = await getStoryData(params);
+
+  return (
+    <StoryblokStory story={data.story} />
+  );
+};
```

Comment:
Finally, just like the homepage, we renders the homepage by using the story data that we just fetched from the Storyblok API